### ISSUE TITLE
Support for exporting Calendar events would be appreciated

### ISSUE THREAD

**Issue by UG-Asger** [Moc/events2bots#2]
For our clan it would be nice that new calendar events were also exported to Discord.

Our needs: New forum posts, New news posts and calendar events

**Comment by wG-Asger** [comment]
It would also be nice to export the live status of servers from LGSL with map name, map image and players, incl. score (and maybe server details).

Just suggestions, don't know if it's possible

**Comment by Moc** [comment]
Hi @wG-Asger, thanks for the feature request. 

First I want to make sure that news, forum, chatbox, and user registrations are properly working. 
Once that's finished, I'll start looking into other areas. 

Separate channels is working already. 

Would you be willing to help me with some testing over the coming weeks? I hope to release a first version this month.

**Comment by Moc** [comment]
As for LGSL, it is unlikely I will integrate this as my events2bots plugin uses event triggers. If I remember correctly, they are not used in the LGSL plugin so that plugin needs to be updated first.

**Comment by UG-Asger** [comment]
Yes, sure. The site is not very active though, but i could do some tests for you
